File: app/src/main/java/com/pkfcooperparry/template/data/remote/ApiEndPoint.java
package com.pkfcooperparry.template.data.remote;


public final class ApiEndPoint {

    public static final String ENDPOINT_LOGIN = ApiClient.BASE_URL + "login";
    public static final String ENDPOINT_USER = ApiClient.BASE_URL + "user";
    public static final String ENDPOINT_LOGOUT = ApiClient.BASE_URL + "logout";

    private ApiEndPoint() {
        //This class is not publicly instantiable
    }
}

File: app/src/main/java/com/pkfcooperparry/template/data/remote/ApiEndPoint.kt
package com.pkfcooperparry.template.data.remote

import com.pkfcooperparry.template.data.remote.ApiClient.Companion.BASE_URL


object ApiEndPoint {

    const val ENDPOINT_LOGIN = """${BASE_URL}login"""
    const val ENDPOINT_USER = """${BASE_URL}user"""
    const val ENDPOINT_LOGOUT = """${BASE_URL}logout"""
}//This class is not publicly instantiable
